data.sql

```diff
@@ -3,4 +3,15 @@ INSERT INTO animals (id, name,date_of_birth,escape_attempts,neutered,weight_kg)
 (1, 'Agumon','2020-02-03',0,true,'10.23'),
 (2, 'Gabumon','2018-11-15',2,true,'8'),
 (3, 'Pikachu', '2021-01-07',1,false, '15.04'),
-(4, 'Devimon', '2017-05-12', 5, true, '11');
\ No newline at end of file
+(4, 'Devimon', '2017-05-12', 5, true, '11');
+
+INSERT INTO animals (id, name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES
```

Comment:
[OPTIONAL]
- I noticed that you're hardcoding values into the `id` column. What do you think about not having to bother about what number is next to be inserted into the `id` column?(imagine you had ten million rows) 😉
You can add the `GENERATED ALWAYS AS IDENTITY` constraint to the `id` column. This ensures that sequential integers that cannot be altered are generated each time a record is being created. I hope it makes sense. 😄 
You can go through this [article](https://www.sqltutorial.org/sql-identity/#:~:text=The%20GENERATED%20ALWAYS%20generates%20sequential,integers%20for%20the%20identity%20column.).

Reply:
Thank you so much for taking time to suggest a better way for me to create an id for each row.